lib/query.js

```diff
@@ -65,6 +65,22 @@ const queryOptionMethods = new Set([
   'wtimeout'
 ]);
 
+const opToThunk = new Map([
```

Comment:
Consider adding a comment to explain this a bit better.

Reply:
Fixed in a41b821